docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -135,40 +134,41 @@ image::create-api-designer-design.png[Image of page to create an API design]
 . Select the design title and click the pen icon. Change the design title to *Orders* and click the *Save changes* icon.
 . Click the *Design* tab, and optionally edit the design as follows:
 .. Provide a version number and a description.
-.. AsyncAPI only: Define terms of service.
+.. _AsyncAPI only:_ Define terms of service.
 .. Add your contact information (name, email address, and URL).
 .. Select a license.
-.. OpenAPI only: Define tags.
+.. _OpenAPI only:_ Define tags.
 .. Define one or more servers.
 .. Configure a security scheme.
-.. OpenAPI only: Specify security requirements.
-.. OpenAPI only: Configure vendor extensions.
+.. _OpenAPI only:_ Specify security requirements.
```

Comment:
Per https://github.com/redhat-developer/app-services-guides/pull/533/files#r903615079, perhaps use parentheses here, too, rather than italics? Just a thought.

Reply:
Done